operators/cv2/imdecode.hpp

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <opencv2/core.hpp>
+#include <opencv2/imgproc.hpp>
+#include <opencv2/imgcodecs.hpp>
+
+#include "ocos.h"
+#include "string_utils.h"
+
+#include <cstdint>
+
+struct KernelImageDecoder : BaseKernel {
+  KernelImageDecoder(const OrtApi& api) : BaseKernel(api) {}
+
+  void Compute(OrtKernelContext* context) {
+    // Setup inputs
+    const OrtValue* const inputs = ort_.KernelContext_GetInput(context, 0ULL);
+    OrtTensorDimensions dimensions(ort_, inputs);
+    if (dimensions.size() != 1ULL) {
+      throw std::runtime_error("Only raw image formats are supported.");
```

Comment:
ORT_CXX_API_THROW is preferred for support NO-Exception mobile build.

Reply:
Done.